net/nginx-util/src/px5g.cpp

```diff
@@ -422,10 +423,13 @@ auto main(int argc, const char ** argv) -> int
 
     catch (const std::exception & e)  {
 
-        auto usage = std::string{"usage: \n"}  ;
-        for (auto cmd : cmds) {
-            usage += std::string{4, ' '} + *argv +" "+ cmd[0] + cmd[1] +"\n";
-        }
+        auto usage = std::accumulate(
+            cmds.begin(),
+            cmds.end(),
+            std::string{"usage: \n"},
+            [=](const auto &use, const auto &cmd)
+            { return use+ std::string{4, ' '} + *argv +" "+cmd[0]+cmd[1]+"\n"; }
```

Comment:
Space after use?

Reply:
Yes, as indent, would be '\t' better?

Reply:
You should probably look into setting up a clang-format file. There's even git support.

Reply:
Ah, I misunderstood. I deleted some spaces here to have linelength<=80 (I did not want to split it on more lines and wanted still to give some parts a bit more space).
But I see that it is irritating and will look into clang-format ...

Reply:
On LKML, Linus Torvalds stated that the line length should be bumped to 100 as it's not the 80s anymore. clang-format has a max line length option.

Reply:
I did reformat it with clang-format (using a custom style based mostly on chromium and Stroustrup). Thanks for the suggestion :-)